docs/src/tutorials/param_estim.md

```diff
@@ -113,7 +113,7 @@ Then finally defining and optimizing using the `PhysicsInformedNN` interface.
 
 ```@example param_estim
 discretization = NeuralPDE.PhysicsInformedNN([chain1, chain2, chain3],
-                                             NeuralPDE.GridTraining(dt), param_estim = true,
```

Comment:
dt is still defined?

Reply:
It seems to be used in other places in this file, thats why I didnt remove it

Reply:
yes, `dt` is used for evenly spaced time grid for the data

Reply:
yes, it can be removed